agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/RuntimeConfigurator.java

```diff
@@ -141,26 +134,6 @@ public void apply(RuntimeConfiguration runtimeConfig) {
       updateSampling(enabled, runtimeConfig.sampling, runtimeConfig.samplingPreview);
     }
 
-    // initialize Profiler
```

Comment:
Not sure this code has to be removed. It seems to be used for Azure function and RP integration. @trask could you confirm that this code should be left?

Reply:
yes, I believe we should keep this for Azure Functions consumption plan